uproot/tree.py

```diff
@@ -546,8 +546,10 @@ def wait():
     def lazyarray(self, branch, interpretation=None, entrysteps=None, entrystart=None, entrystop=None, flatten=False, awkwardlib=None, cache=None, basketcache=None, keycache=None, executor=None, persistvirtual=False):
```

Comment:
(I'm looking at this on a laptop, now.) The `TTree.lazyarray` method should have and pass on a `chunked=True` parameter, too, right?

There's `TTree.lazyarray` (singular), `TTree.lazyarrays` (plural), and `TBranch.lazyarray` (singular). As you explained, the `uproot.lazyarray` and `uproot.lazyarrays` can't be unchunked (unless, of course, you `awkward.concatenate` them, but that's not obviously helpful. (Multi-file arrays might always or almost always be too big, unchunked.) And `uproot.daskarray` and `uproot.daskframe` shouldn't have this option because they _must_ be chunked.